coalahtml/_coalahtml/index.html

```diff
@@ -9,9 +9,9 @@
     <link rel='stylesheet' type='text/css'
       href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext'>
     <link rel="stylesheet"
-          href="bower_components/bootstrap/dist/css/bootstrap.css">
+    href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
```

Comment:
@AbdealiJK  Well. We can have css files offline. That'd work offline and would also decrease latency while online. What say? Cons: It'd just add a few Kbs. :P